### PR TITLE
Import catalog and common less files after patternfly to allow overri…

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
…des.

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
@rhamilto Good idea, added comments.

File: app/styles/main.less [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
-
 // PatternFly
 @import "@{pf-less-path}/patternfly.less";
 @import "@{pf-less-path}/patternfly-additions.less";
 
+// origin-web-catalog - after patternfly to gain any overrides
+@import "@{catalog-less-path}/catalog.less";
+
+// origin-web-common - after patternfly to gain any overrides
+@import "@{common-less-path}/common.less";
```
Comment (rhamilto) [inline app/styles/main.less]: Thinking about this further, maybe catalog should come last so we could override common without the need to cut releases of both?

Comment (jeff-phillips-18) [inline app/styles/main.less]: Yeah, makes sense. Done.

**Comment by spadgett** [comment]
[merge]

**Pull request merged**